src/app/systelab-components/applicationframe/sidebar/app-sidebar.component.ts

```diff
@@ -3,7 +3,7 @@ import { DialogService } from '../../modal/dialog/dialog.service';
 
 export class ApplicationSidebarTab {
 
-	constructor(public id: string, public name: string, public isSelected: boolean, public subMenu?: ApplicationSidebarTab[]) {
+	constructor(public id: string, public name: string, public isSelected: boolean, public subMenu?: ApplicationSidebarTab[], public action?: any) {
```

Comment:
Change `public action?: any` to `public action?: (id: string) => void`

Reply:
Done